pkg/graph/schema.graphqls

```diff
@@ -189,16 +189,16 @@ type UserStudentConnection {
 }
 
 enum UserOrderBy {
```

Comment:
I don't remember seeing the frontend updated for the sort key changes (may have missed it, or there's nothing to update). Is this change also reflected in the frontend?

Reply:
Yes